Router/Router/View/ResetPasswordViewController.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  RecoveryPasswordViewController.swift
```

Comment:
```suggestion
//  ResetPasswordViewController.swift
```
Не совпадает название класса/файла и описание